src/data/maps.json

```
@@ -1439,6 +1439,130 @@
+                        "bottom": -4,
+                        "top": -2
+                    },
+                    {
+                        "position": [326, -118.5],
+                        "text": "Swamp",
+                        "bottom": -4,
+                        "top": -2
+                    },
+                    {
+                        "position": [418.4, 118],
+                        "text": "Village",
+                        "bottom": -4,
+                        "top": -2
+                    },
+                    {
+                        "position": [288, 144],
+                        "text":"Cabins",
+                        "bottom": -4,
+                        "top": -2
+                    },
+                    {
+                        "position": [128, 93],
+                        "text": "Cottages",
+                        "bottom": -4,
+                        "top": -2
+                    },
+                    {
+                        "position": [-355, 188],
+                        "text": "Tank Bridge",
+                        "bottom": -4,
+                        "top": -2
+                    },
+                    {
+                        "position": [-338.6, 525],
+                        "text": "Pier",
+                        "bottom": -4,
+                        "top": -2
+                    },
+                    {
+                        "position": [216, 424],
+                        "text": "Scav Island",
+                        "bottom": -4,
+                        "top": -2
+                    },
+                    {
+                        "position": [-96, -6],
+                        "text": "Bus Stop",
+                        "bottom": -4,
+                        "top": -2
+                    },
+                    {
+                        "position": [52 , 134],
+                        "text": "Construction",
+                        "bottom": -4,
+                        "top": -2
+                    },
+                    {
+                        "position": [-153, -290],
+                        "text": "Bunker",
+                        "bottom": -4,
+                        "top": -2
+                    },
+                    {
+                        "position": [-625, 484],
+                        "text": "Crane",
+                        "bottom": -4,
+                        "top": -2
+                    },
+                    {
+                        "position": [-622, -202],
+                        "text": "Scav Farm",
+                        "bottom": -4,
+                        "top": -2
+                    },
+                    {
+                        "position": [-171, -83],
+                        "text": "West Wing",
+                        "bottom": 0,
+                        "top": 3
+                    },
+                    {
+                        "position": [-329, -85],
```

Height should be the same as West Wing
```suggestion
                        "position": [-329, -83],
```